ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -430,18 +439,20 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
 
     private func updateTitleViewFont() {
         if let window = window {
-            let titleFont = tokenSet[.textFont].uiFont
+            let titleFont = attributedTitleFont != nil ? attributedTitleFont : tokenSet[.textFont].uiFont
```

Comment:
To avoid having to do these nil checks, can we just initialize 
```
attributedTitleFont = attributes[NSAttributedString.Key.font] as? UIFont ?? tokenSet[.textFont].uiFont
attributedTitleColor = attributes[NSAttributedString.Key.foregroundColor] as? UIColor ?? tokenSet[.textColor].uiColor
```
Then we can just pass in `attributedTitleFont` and `attributedTitleColor`?

Reply:
Since `attributedTitleFont` and `attributedTitleColor` are initialized in `setup`, wouldn't that be a little confusing here? That `titleFont` is set to `attributedTitleFont`?

Reply:
`attributedTitleFont` and `attributedTitleColor` would need to be renamed to something like `resolvedFont` and `resolvedTitleColor`.

Reply:
Agreed -- I'd much rather we have a single source of truth, at least internally, to avoid having to sprinkle `nil` checks around. That's gonna be super error-prone going forward, I fear.

Reply:
Thanks for the suggestion, that makes sense alright so I will update it!

Reply:
At present leaving the vars as optional so I don't need to assign values in as many places, but if we want them non-optional I can make sure they are set on init

Reply:
Another option could be to make the `lazy`, and init them with the `tokenSet`, then overwrite that with the values from the attributed string if needed.

Reply:
Thanks, I think that would end up with similar additional code, so I might just go with the current iteration for now...